src/routing/Outlet.ts

```diff
@@ -37,8 +37,8 @@ export const Outlet = factory(function Outlet({
 			if (handle) {
 				cache.set('handle', handle);
 			}
-			invalidator();
 		}
+		invalidator();
```

Comment:
This needs to `invalidate` every render, but not actually queue an invalidation just mark the widget as dirty.